### DISCUSSION TITLE
23.12.12 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#87] [Daily Scrum]
.

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - PR 올리기
- 내일은 어떤 일을 하나요?
    - 개발한 기능 리드미 작성

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - Artist관련 테스트 코드 추가, 수정
- 내일은 어떤 일을 하나요?
  - 코드 정리
  - 문서화

**Comment by Nine-JH** [comment]
* 오늘은 어떤 일을 했나요?
  * 최적화 및 통합 테스트 완료
* 내일은 어떤 일을 하나요?
  * 내일은 사정이 있어서 작업을 못할 것 같네요
